### PR TITLE
Creates Docs for Google Cloud Storage Source

### LINKED ISSUE #118: Add documentation for google storage source

### PR DESCRIPTION
**Pull request by JeffNeff**
Closes #118

### CODE REVIEW

**Comment by JeffNeff** [comment]
relates to https://github.com/triggermesh/frontend/issues/1750

File: docs/sources/googlecloudstorage.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "updated": "2021-03-26T19:13:09.443Z",
+    "storageClass": "STANDARD",
+    "timeStorageClassUpdated": "2021-03-26T19:13:09.443Z",
+    "size": "1716402",
+    "md5Hash": "EDETqCAJ5kWxYRL7X/2ykw==",
+    "mediaLink": "https://www.googleapis.com/download/storage/v1/b/tmdemo/o/wp5960142-the-great-wave-off-kanagawa-hd-wallpapers.jpg?generation=1616785989346461&alt=media",
+    "crc32c": "G3Ynzg==",
+    "etag": "CJ3JypTVzu8CEAE="
+  }
+```
```
Comment (antoineco) [inline docs/sources/googlecloudstorage.md:62]: We decided against having that in documentation pages in #109.

Please see @daceynolan's templates in https://github.com/triggermesh/docs/tree/master/templates/source.

**Review by antoineco — CHANGES_REQUESTED** [review]
The format is very inconsistent with the template we agreed to use.
Let me know if you want me to take over.

**Comment by JeffNeff** [comment]
@antoineco all yours :)

**Pull request merged**